tests/integration/reqs/test_submit_multisigned.py

```diff
@@ -59,7 +65,7 @@ async def test_basic_functionality(self, client):
         issuer = Wallet.create()
         tx = TrustSet(
             account=WALLET.classic_address,
-            sequence=WALLET.sequence,
+            sequence=await get_next_valid_seq_number(WALLET.classic_address, client),
```

Comment:
Does the wallet even need to be here, if it's filled in further down?

Reply:
Not filled later down. Also, this test case is being changed drastically in multisign PR.

Reply:
Whoops, I meant sequence here, not wallet

Reply:
Sequence is not filled later on in this test. Before multisign PR gets merged, autofill isn't quite compatible with multisign.